feature_engine/selection/drop_psi_features.py

```diff
@@ -451,11 +445,11 @@ def _split_dataframe(self, X):
                 "dataframe."
             )
 
-        # If cut_off is not pre-defined, compute it.
+        # If cut_off is not pre-defined, compute it and set it as attribute.
         if not self.cut_off:
-            cut_off = self._get_cut_off_value(reference)
-        else:
-            cut_off = self.cut_off
+            self.cut_off = self._get_cut_off_value(reference)
```

Comment:
we can't override parameters set by the user. That is, by sklearn convention. The transformer would not pass the check_estimator tests like this. 

This is why I had a temporal variable cut_off.

Reply:
I learned that the hard way. Now I created a self.cut_off_ attribute so that the value of the cut_off is available to the user when the cut_off is not explicitly available.

Reply:
A quick note, if the class has the new attribute `self.cut_off_`, it also needs  to exist when the user defines cut_off themselves, in which case I guess, it would be the same value. In other words, the class should always have that parameter and not just when cut_off is None.

Reply:
Yes, that is howe is it currently designed. self.cut_off_ is either assigned to the cut_off defined in the init or computed.
![image](https://user-images.githubusercontent.com/32060943/140310261-4af5e7b3-b9c6-43af-b949-ec4d41e60de8.png)